assets/src/components/map/controls/StreetViewSwitch.tsx

```diff
@@ -109,5 +109,10 @@ export const StreetViewControl = ({
     </>
   )
 
-  return portalElement ? ReactDOM.createPortal(control, portalElement) : null
+  return (
+    <>
+      {portalElement && ReactDOM.createPortal(control, portalElement)}
+      {streetViewEnabled && <CutoutOverlay.FollowMapMouseMove />}
```

Comment:
I was fairly confident that this belongs with the `StreetViewSwitch` component so that the `StreetViewSwitch` is encapsulated but I'll have to come back to it after thinking on it.